JumpscaleLibs/sal/zosv2/resource.py

```diff
@@ -65,7 +69,15 @@ def calculate_used_resources_cost(self):
 
         return resource_units_per_node
 
-    def payout_farmers(self, tfchain_wallet, resource_units_per_node, reservation_id):
+    def payout_farmers(self, client, resource_units_per_node, reservation_id):
+        if client._classname == "tfchainwallet":
```

Comment:
@zaibon is there any other (cleaner) way to check what kind of client is passed?

Reply:
to be very sure I would use [isinstance](https://docs.python.org/3.6/library/functions.html#isinstance) so you can be exactly sure of the type your are dealing with.